### DISCUSSION TITLE
Image File Size

### DISCUSSION THREAD

**Discussion by Churros396** [opengisch/QField#5413] [Q&A]
Hello,

we are capturing one to one images in our project. Overtime now the sync load is increasing due to the number of images. I was wondering if adding a zip archive as photo directory would work, especially while using the Cloud. Other tips on how you manage your photos would be great. 

Thanks for the input.

Cu around

**Comment by BTreeTech** [comment]
JPGs are already compressed so not sure how much you gain by zipping them. In our system we have reduced the image size to suit the end use which which is much smaller than the default full resolution so saves loads of space.
